pkg/runtime/runtime.go

```diff
@@ -1459,7 +1457,7 @@ func isEnvVarAllowed(key string) bool {
 	}
 
 	// If we have a `DAPR_ENV_KEYS` env var (which is added by the Dapr Injector in Kubernetes mode), use that as allowlist too
-	allowlist := strings.ToUpper(os.Getenv(securityConsts.EnvKeysEnvVar))
+	allowlist := os.Getenv(securityConsts.EnvKeysEnvVar)
```

Comment:
THis really needs not to be reverted please

```suggestion
	allowlist := strings.ToUpper(os.Getenv(securityConsts.EnvKeysEnvVar))
```